readthedocs/subscriptions/managers.py

```diff
@@ -43,25 +45,7 @@ def get_or_create_default_subscription(self, organization):
             )
             return None
 
-        stripe_customer = get_or_create_stripe_customer(organization)
-        stripe_subscriptions = stripe_customer.subscriptions.exclude(
-            status=SubscriptionStatus.canceled
-        )
-        if stripe_subscriptions.count() > 1:
-            log.warning(
-                "Customer with more than one active subscription.",
-                stripe_customer=stripe_customer.id,
-            )
```

Comment:
we could still leave this warning (on subscriptions/utils.py where this code was moved), but I think it makes more sense to query all of our subs searching for cases like this and decide what to do if any.

Reply:
I think I'm fine trusting that our data is consistent. However, if it's not, how we will detect this? This code at least gave us some hint were to look at when Stripe data was inconsistent.

We added it to detect these cases because we weren't sure which ones were those customers.

Reply:
We have 0 cases of more than one active subs

```
In [11]: from djstripe import models as djstripe
    ...: from django.db import models
    ...: 
    ...: subquery = models.Subquery(
    ...:     djstripe.Subscription.objects
    ...:     .filter(customer=models.OuterRef("id"), status="active")
    ...:     .annotate(count=models.Count("id"))
    ...:     .values("count")
    ...: )
    ...: 
    ...: q = djstripe.Customer.objects.annotate(active_subs_count=subquery).filter(active_subs_count__gt=1)

In [12]: q.count()
Out[12]: 0

```

Should we still have this warning?

Reply:
Same on .org

Reply:
I'm merging this, so I can create a new PR with some changes I have, with main as base instead of this branch as base. Let me know if you still think we should log this, I can just open a new PR for that.